test/v2/models/response/retrieveBalanceAndTransactions/FinancialDetailsItemSpec.scala

```diff
@@ -42,70 +42,127 @@ class FinancialDetailsItemSpec extends UnitSpec with FinancialDetailsItemFixture
         }
       }
 
-      "converting outgoingPaymentMethod" must {
-        def json(value: String): JsValue                       = Json.parse(s"""{ "outgoingPaymentMethod": "$value" }""")
-        def model(value: Option[String]): FinancialDetailsItem = financialDetailsItemModelEmpty.copy(outgoingPaymentMethod = value)
+      "converting clearingReason" must {
+        def json(value: String): JsValue = Json.parse(s"""{ "clearingReason": "$value" }""")
+
+        def model(value: Option[String]): FinancialDetailsItem = financialDetailsItemModelEmpty.copy(clearingReason = value)
 
         "convert if present and a mapping is defined" when {
           def doTest(downstreamValue: String, mtdValue: String): Unit =
             s"downstream value is $downstreamValue" in {
               json(downstreamValue).as[FinancialDetailsItem] shouldBe model(Some(mtdValue))
             }
 
-          Seq("A" -> "Repayment to Card", "P" -> "Payable Order Repayment", "R" -> "BACS Payment out").foreach((doTest _).tupled)
+          Seq("01" -> "Incoming Payment", "02" -> "Outgoing Payment", "05" -> "Reversal", "06" -> "Manual Clearing", "08" -> "Automatic Clearing")
+            .foreach((doTest _).tupled)
         }
 
         "leave absent if present and no mapping is defined" in { // FIXME is this right?
```

Comment:
I wonder if it might be worth adding a mapping for "Write Off" and then anything that does not have a mapping can default to "Other". That should cover all possible cases (assuming req spec is correct):
<img width="259" alt="image" src="https://user-images.githubusercontent.com/106095575/193621226-da3cf535-cba0-4776-b54e-2625139aabc5.png">
Alternatively, if current test is fine, can we remove the FIXME comment?